client/src/app/site/pages/meetings/pages/assignments/modules/assignment-poll/components/assignment-poll-detail-content/assignment-poll-detail-content.component.html

```diff
@@ -70,6 +70,90 @@
+                                            {{ getVoteAmount(vote, row) | pollPercentBase: poll:row }}
+                                        </span>
+                                    </div>
+                                </td>
+                                <td>
+                                    <div class="result">
+                                        <span>
+                                            {{ getVoteAmount(vote, row) | pollParseNumber }}
+                                        </span>
+                                    </div>
+                                </td>
+                            </tr>
+                        </ng-container>
+                        <ng-container *ngIf="row.class !== 'user'">
+                            <tr *ngFor="let vote of filterRelevantResults(row.value)">
+                                <td>
+                                    {{ row.votingOption | pollKeyVerbose | translate }}
+                                </td>
+                                <td>
+                                    <div class="result" [ngClass]="getVoteClass(vote)">
+                                        <span *ngIf="vote.showPercent">
+                                            {{ getVoteAmount(vote, row) | pollPercentBase: poll:row }}
+                                        </span>
+                                    </div>
+                                </td>
+                                <td>
+                                    <div class="result">
+                                        <span>
+                                            {{ vote.amount | pollParseNumber }}
+                                        </span>
+                                    </div>
+                                </td>
+                            </tr>
+                        </ng-container>
+                    </ng-container>
+                    <tr *ngIf="isPercentBaseEntitled && poll.entitled_users_at_stop" class="entitled-users-row">
+                        <td>{{ 'Entitled users' | translate }}</td>
+                        <td class="result">
+                            <div>
+                                <span>
+                                    {{ poll.entitled_users_at_stop.length | pollPercentBase: poll }}
+                                </span>
+                            </div>
+                        </td>
+                        <td>
+                            <div class="result">
+                                <span>
+                                    {{ poll.entitled_users_at_stop.length }}
+                                </span>
+                            </div>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+            <!-- Chart -->
+            <div class="doughnut-chart">
+                <os-chart type="doughnut" [data]="chartData"></os-chart>
+            </div>
+        </div>
+    </div>
+
```

Comment:
Is this the same code as in `motion-poll-detail-content`? If so, please make an own file/component for that view. Otherwise, it is good.

Reply:
I think it may be a bit different, but if they're similar it's a good idea, I'll look into it